apps/web/components/InfoCardGrid/InfoCardGrid.tsx

```
@@ -0,0 +1,49 @@
+  useEffect(() => {
+    if (width < theme.breakpoints.md) {
+      return setIsMobile(true)
+    }
+    setIsMobile(false)
+  }, [width])
+
+  if (columns === 1 || isMobile) {
+    return (
+      <Stack space={3}>
+        {cards.map((c) => (
+          <InfoCard variant={variant} size={'large'} {...c} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)